lib/themes/dosomething/paraneue_dosomething/scss/patterns/_photo.scss

```diff
@@ -23,7 +23,7 @@
       padding: ($base-spacing / 2);
     }
 
-    img {
+    .wrapper img {
```

Comment:
Let's move this `img { ... }` to be nested under the `> .wrapper { ... }` rule above. The `>` is important whenever targeting `.wrapper` since we use "wrapper" as a standard "child" class in a bunch of different patterns.